src/framework/parsers/glb-parser.js

```diff
@@ -1804,17 +1804,17 @@ const createScenes = function (gltf, nodes) {
     return scenes;
 };
 
-const createCameras = function (gltf, nodes, options) {
+const createCameras = (gltf, nodes, options) => {
 
     let cameras = null;
 
     if (gltf.hasOwnProperty('nodes') && gltf.hasOwnProperty('cameras') && gltf.cameras.length > 0) {
 
-        const preprocess = options && options.camera && options.camera.preprocess;
-        const process = options && options.camera && options.camera.process || createCamera;
-        const postprocess = options && options.camera && options.camera.postprocess;
+        const preprocess = options?.camera?.preprocess;
+        const process = options?.camera?.process || createCamera;
```

Comment:
Do we want to standardize on `??` instead of `||` for these kinds of use cases? Which means 'if the left side is null or undefined, use the right side' (rather than a logical op).

Reply:
Yeah updating to ?? would make sense. I have a followup PR already which is based on this one and will update to ?? there (since some have been removed).